recipe/meta.yaml

```diff
@@ -23,6 +23,7 @@ requirements:
     - pip
```

Comment:
as here is pip, wheel and setuptools are still added ... agree that hatchling is actually used, but implicit those two are still added by conda-build